Resources/public/js/models/ez-locationmodel.js

```diff
@@ -178,6 +178,72 @@ YUI.add('ez-locationmodel', function (Y) {
                 location.reset();
                 callback(error, response);
             });
+        },
+
+        /**
+         * Loads path of the current location. The result is the array containing
+         * eZ.Location objects present on the path. The array doesn't contain current location.
+         *
+         * @method loadPath
+         * @param {Object} options
+         * @param {Object} options.api the JS REST client instance
+         * @param {Function} callback
+         * @param {false|Error} callback.error
+         * @param {Array} callback.locations the array of Locations
+         */
+        loadPath: function (options, callback) {
+            var locations = [],
+                that = this;
+
+            this._loadAncestors(options, function (error, response) {
+                var hits;
+
+                if (error) {
+                    callback(error);
+                    return;
+                }
+
+                hits = response.document.View.Result.searchHits.searchHit;
+
+                Y.Array.each(hits, function (hit) {
+                    if (hit.value.Location._href !== this.get('id')) {
+                        var location = new Y.eZ.Location();
+
+                        location.setAttrs(location.parse({document: hit.value}));
+                        locations.push(location);
+                    }
+                }, that);
+
+                locations.sort(function (a, b) {
```

Comment:
Note: that's a workaround because the REST views do not support sorting yet (see [EZP-24315](https://jira.ez.no/browse/EZP-24315), so you should create an improvement in JIRA blocked by EZP-24315 about the removal of this code when EZP-24315 is implemented.